clientconfig/clientconfig.go

```diff
@@ -0,0 +1,28 @@
+package clientconfig
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/Clever/kayvee-go/v7/logger"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+func ClientLogger(wagAppName string) *logger.Logger {
+	return logger.NewConcreteLogger(fmt.Sprintf("%s-wagclient", wagAppName))
+}
+
+func WithoutTracing(wagAppName string) (*logger.Logger, *http.RoundTripper) {
+	return ClientLogger(wagAppName), &http.DefaultTransport
+}
+
+func WithTracing(ctx context.Context, wagAppName string, exporter sdktrace.SpanExporter) (*logger.Logger, *http.RoundTripper) {
```

Comment:
Needed to add `ctx` since it seems that is required for `DefaultInstrumentor()`

Reply:
Discussed [here](https://github.com/Clever/wag/pull/454#discussion_r1091055522) that we don't need `ctx` anymore, which is nice